tests_config/setup.js

```diff
@@ -0,0 +1,7 @@
+const runSpec = require('./run_spec.js');
```

Comment:
interesting, it looks like from this file on GH is no longer formatting js? 🤷‍♂️

Reply:
Looks normal in my screen.